bin/runalyzer-gather

```diff
@@ -1,9 +1,6 @@
 #! /usr/bin/env python
```

Comment:
Is there a reason to leave this as `python` and not `python3`?

Reply:
I think [PEP 394](https://www.python.org/dev/peps/pep-0394/) suggests `python` if it's primarily used within a conda/virtual env, otherwise prefer the more specific `python3`

Reply:
9fcebf4. And thanks, TIL!